images/nginx/rootfs/build.sh

```diff
@@ -20,42 +20,42 @@ set -o pipefail
-export OPENTRACING_CPP_VERSION=1.6.0
+export SETMISC_VERSION=fddc347134e97d22c34dc5189e76b0ec97bf84d7
+export MORE_HEADERS_VERSION=f85af9649b858e21b400a2150a4c7b8ebd36e921
+export NGINX_DIGEST_AUTH=1.0.0
+export NGINX_SUBSTITUTIONS=b8a71eacc7f986ba091282ab8b1bbbc6ae1807e0
+export NGINX_OPENTRACING_VERSION=2d9f40fa86380b019698f131a803f415ad24c033
+export OPENTRACING_CPP_VERSION=f86b33f3d9e7322b1298ba62d5ffa7a9519c4c41
 export ZIPKIN_CPP_VERSION=f69593138ff84ca2f6bc115992e18ca3d35f344a
-export YAML_CPP_VERSION=yaml-cpp-0.6.3
-export JAEGER_VERSION=0.7.0
-export MSGPACK_VERSION=3.2.1
-export DATADOG_CPP_VERSION=7b560e5c13324c0581476dad3bd8ac4ac5f64045
-export MODSECURITY_VERSION=22e53aba4e3ae8c7d59a3672d6727e49246afe96
-export MODSECURITY_LIB_VERSION=v3.0.4
-export OWASP_MODSECURITY_CRS_VERSION=v3.3.0
-export LUA_NGX_VERSION=138c1b96423aa26defe00fe64dd5760ef17e5ad8
-export LUA_STREAM_NGX_VERSION=0.0.9
-export LUA_UPSTREAM_VERSION=0.07
-export LUA_CJSON_VERSION=2.1.0.8
+export YAML_CPP_VERSION=b591d8ae2ad1ff373273c3e05973adf6c46abfa8
+export JAEGER_VERSION=e90642a250289623fc13f650991d108e9a19b6b1
+export MSGPACK_VERSION=3.3.0
+export DATADOG_CPP_VERSION=5e3e3571d2341883a15fc4b5b443f930cc70e8c1
+export MODSECURITY_VERSION=2497e6ac654d0b117b9534aa735b757c6b11c84f
+export MODSECURITY_LIB_VERSION=v3.0.5
+export OWASP_MODSECURITY_CRS_VERSION=v3.3.2
+export LUA_NGX_VERSION=66a4d417d654404ade32fcc600ff1381e5e02f9c
+export LUA_STREAM_NGX_VERSION=b977cee61a1cc0ea988a581cf0adbc955c9a95ab
+export LUA_UPSTREAM_VERSION=8aa93ead98ba2060d4efd594ae33a35d153589bf
+export LUA_CJSON_VERSION=4b350c531de3d71008c77ae94e59275b8371b4dc
 export NGINX_INFLUXDB_VERSION=5b09391cb7b9a889687c0aa67964c06a2d933e8b
-export GEOIP2_VERSION=3.3
-export NGINX_AJP_VERSION=bf6cd93f2098b59260de8d494f0f4b1f11a84627
-
-export LUAJIT_VERSION=2.1-20201027
-
-export LUA_RESTY_BALANCER=af4508f7aa5560c7d810922c2515b557f9e5d51a
-export LUA_RESTY_CACHE=0.10
-export LUA_RESTY_CORE=0.1.21
-export LUA_RESTY_COOKIE_VERSION=766ad8c15e498850ac77f5e0265f1d3f30dc4027
-export LUA_RESTY_DNS=0.21
-export LUA_RESTY_HTTP=0.15
-export LUA_RESTY_LOCK=0.08
-export LUA_RESTY_UPLOAD_VERSION=0.10
-export LUA_RESTY_STRING_VERSION=0.12
-export LUA_RESTY_MEMCACHED_VERSION=0.15
-export LUA_RESTY_REDIS_VERSION=0.29
-export LUA_RESTY_IPMATCHER_VERSION=1a0a1c58fd085b15eedee58de8b5f45c27aff7bc
+export GEOIP2_VERSION=a26c6beed77e81553686852dceb6c7fdacc5970d
+export NGINX_AJP_VERSION=a964a0bcc6a9f2bfb82a13752d7794a36319ffac
+
+export LUAJIT_VERSION=2.1-20210510
+
+export LUA_RESTY_BALANCER=2e1848e729eb8558d533ce97ffdf61edead39953
+export LUA_RESTY_CACHE=dacd022b4b8e3a73ed25886f2f9db043075ddb31
+export LUA_RESTY_CORE=b8a4b2cfe35bcb65c2ada52c17503ece7cd3288a
+export LUA_RESTY_COOKIE_VERSION=303e32e512defced053a6484bc0745cf9dc0d39e
+export LUA_RESTY_DNS=0.22
+export LUA_RESTY_HTTP=0.16.1
+export LUA_RESTY_LOCK=ad94745a4731ee6ddbffbc3b602df25ca7c6f7ba
+export LUA_RESTY_UPLOAD_VERSION=7baca92c7e741979ae5857989bbf6cc0402d6126
```

Comment:
We can keep 0.10 here -> https://github.com/openresty/lua-resty-upload/compare/v0.10...master